mmeds/validate.py

```diff
@@ -421,7 +422,9 @@ def run(self):
         else:
             if self.metadata_type == 'subject':
                 tables = fig.SUBJECT_TABLES
-                subjects = self.df['Subjects']
+                if 'Subjects' in self.df.keys():
+                    #Only define subjects if subject table is correctly uploaded
+                    subjects = self.df['Subjects']
```

Comment:
If subjects aren't correctly uploaded, should an error be thrown here?

Reply:
An error gets thrown later in a way that lets the user reupload the file. This was causing a 505 error and stopping everything